client/api/src/main/java/io/smallrye/graphql/client/typesafe/api/Multiple.java

```diff
@@ -25,9 +25,9 @@
  *     Bar bar;
  * }
  * </pre>
- *
+ * <p>
```

Comment:
Should this not be </p> ?

Reply:
Maybe your formatting got mixed up, so I'm not sure what you meant. Maybe this is better?